backend/maelstro/core/clone.py

```diff
@@ -47,6 +47,87 @@ def gn_dst(self) -> GnApi:
     def gs_dst(self) -> RestService:
         return self.geo_hnd.get_gs_service(self.dst_name, is_source=False)
 
+    def involved_resources(
+        self,
+        copy_meta: bool,
+        copy_layers: bool,
+        copy_styles: bool,
+    ) -> dict[str, list[dict[str, Any]]]:
+        self.copy_meta = copy_meta
+        self.copy_layers = copy_layers
+        self.copy_styles = copy_styles
+
+        with get_georchestra_handler() as geo_hnd:
+            self.geo_hnd = geo_hnd
+
+            zipdata = self.gn_src.get_record_zip(self.uuid).read()
+            self.meta = Meta(zipdata)
+
+            resources: dict[str, list[dict[str, Any]]] = {
+                "metadata": [],
+                "data": [],
+            }
+
+            src_gn_info = self.geo_hnd.get_service_info(
+                self.src_name, is_source=True, is_geonetwork=True
+            )
+            src_gn_url = src_gn_info["url"]
+            dst_gn_info = self.geo_hnd.get_service_info(
+                self.dst_name, is_source=False, is_geonetwork=True
+            )
+            dst_gn_url = dst_gn_info["url"]
+
+            resources["metadata"].append(
+                {
+                    "src": src_gn_url,
+                    "dst": dst_gn_url,
+                    "metadata": (
+                        [
+                            {
+                                "title": self.meta.get_title(),
+                            }
+                        ]
+                        if self.copy_meta
+                        else []
+                    ),
+                }
+            )
+
+            dst_gs_info = self.geo_hnd.get_service_info(
+                self.dst_name, is_source=False, is_geonetwork=False
+            )
+            dst_gs_url = dst_gs_info["url"]
+
+            geoservers = self.meta.get_gs_layers(config.get_gs_sources())
+            for server_url, layer_names in geoservers.items():
```

Comment:
I'd suggest not to merge all the elements of different geoservers but keep the structure per geoserver URL, that will probably also be nicer for the user.

Reply:
OK, alright. I got lost with the constant dst_gs_url in each list element